fabrictestbed_extensions/__init__.py

```diff
@@ -1,3 +1,5 @@
+# Please update version number here when making a release: flit, the
+# build backend we use, picks up the version from here.
```

Comment:
This comment seems to be a merge artifact.  It is not true anymore, because all package metadata is now in `pyproject.toml`. Can we remove this?